packages/suite/src/components/suite/modals/ReduxModal/ConfirmAddressModal.tsx

```diff
@@ -5,13 +5,20 @@ import { Translation } from 'src/components/suite';
 import { useDisplayMode, useSelector } from 'src/hooks/suite';
 import { selectSelectedAccount } from 'src/reducers/wallet/selectedAccountReducer';
 import { ConfirmValueModal, ConfirmValueModalProps } from './ConfirmValueModal/ConfirmValueModal';
+import { CryptoSymbol } from 'invity-api';
 
 interface ConfirmAddressModalProps
     extends Pick<ConfirmValueModalProps, 'isConfirmed' | 'onCancel' | 'value'> {
     addressPath: string;
+    receiveSymbol?: CryptoSymbol;
```

Comment:
I am not sure with `CryptoSymbol` here as we use symbols only from `NetworkSymbol` in Trezor Suite.

CryptoSymbol value should always be available in NetworkSymbol. In case there is different symbol, then Trezor cannot show valid address

I would change it to `NetworkSymbol` if possible

Reply:
Done.

After a discussion with @tomasklim I have made two updates:
- `useSelector` was moved to the children component
- for cryptocurrencies as `USDT@ETH` was added better translation - `USDT receive address on ETH network`